src/main/java/library/typeOfOnlineLibrary/people/customer/Customer.java

```diff
@@ -0,0 +1,24 @@
+package library.typeOfOnlineLibrary.people.customer;
+
+import library.typeOfOnlineLibrary.people.stuff.People;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public  class Customer extends People {
+    private Map<String, String> books = new HashMap<String, String>();
+
+    private long passportData;
+    @Override
+    protected void setName(String name){
+     name =this.name;
+    }
+    @Override
+   protected void setSex(boolean sex){
```

Comment:
let's rename to 'setGender()'

Reply:
regarding to java convention boolean vars have to start with 'is' or 'has' ... 
i.e. isFemale, isMale, hasAttribute()...etc.

Reply:
FIXED